Action.swift

```diff
@@ -18,6 +18,11 @@ public final class Action<Input, Element> {
     public let _enabledIf: Observable<Bool>
     public let workFactory: WorkFactory
 
+    /// Inputs that execute the action.
+    /// Inputs via execute() also appear in this subject.
+    public let inputs = PublishSubject<Input>()
```

Comment:
I think input values should be appear in this subject regardless of enabled so that inputs count equals elements count + errors count. And I think this should be documented here.